src/actions/login.ts

```
@@ -0,0 +1,37 @@
+    const { setUser } = useUserStore.getState()
+    const { setAuth } = useAuthStore.getState()
+
+    const { user, ...tokens } = response.data
+
+    setUser(user)
+    setAuth(tokens)
+    Cookie.set('isAuthenticated', 'true')
+
+    return
+  } catch (error) {
+    throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)